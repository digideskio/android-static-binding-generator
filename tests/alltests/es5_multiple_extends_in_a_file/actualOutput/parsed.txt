android.widget.Button22222222**onClick222222,onClick222222222*my.custom.ClassFromOrigina2222222222
android.widget.Button333333**onClick33333,onClick333333333*my.custom.ClassFromOrigina3333333333
android.widget.Button1111111111*_fparsed_l2_c52__ClassFromOrigina1111111*onClick111111*
android.widget.Button44444444*_fparsed_l25_c50__ClassFromOrigina44444*onClick4444,onClick4444444*