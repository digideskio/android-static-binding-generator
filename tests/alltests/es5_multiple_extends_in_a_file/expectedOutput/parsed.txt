EXTEND_CLASS: android.widget.Button1111111111 - EXTEND_HASH:  - OVERRIDDEN_METHODS: onClick111111,onClick111111111 - JAVA_FILE: my.custom.ClassFromOrigina1111111
EXTEND_CLASS: android.widget.Button22222222 - EXTEND_HASH:  - OVERRIDDEN_METHODS: onClick222222,onClick222222222 - JAVA_FILE: my.custom.ClassFromOrigina2222222222
EXTEND_CLASS: android.widget.Button333333 - EXTEND_HASH:  - OVERRIDDEN_METHODS: onClick33333,onClick333333333 - JAVA_FILE: my.custom.ClassFromOrigina3333333333
EXTEND_CLASS: android.widget.Button44444444 - EXTEND_HASH:  - OVERRIDDEN_METHODS: onClick4444,onClick4444444 - JAVA_FILE: my.custom.ClassFromOrigina44444

